packages/expo-dev-menu/src/DevMenu.ts

```diff
@@ -0,0 +1,7 @@
+import { NativeModules } from 'react-native';
+
+const { ExpoDevMenu } = NativeModules;
+
+export function openMenu(): void {
+  ExpoDevMenu.openMenu();
+}
```

Comment:
I wish we could grab some feedback here:
```suggestion
export function openMenuAsync(): void {
  return await ExpoDevMenu.openMenuAsync();
}
```